examples/native/native_test.c

```diff
@@ -1221,16 +1215,9 @@ int TPM2_Native_Test(void* userCtx)
         cmdIn.create.inSensitive.sensitive.userAuth.size);
     cmdIn.create.inPublic.publicArea.type = TPM_ALG_SYMCIPHER;
     cmdIn.create.inPublic.publicArea.nameAlg = TPM_ALG_SHA256;
-#ifdef WOLFTPM_MCHP
-    /* workaround for issue with both sign and decrypt being set */
```

Comment:
marking this for comment in the PR about EncryptDecrypt, TPMA_OBJECT_Sign/decrypt and MCHP

Reply:
I retested with all module and it did not need both sign/decrypt set. So this seems like a good change to make for all.